lib/models/addon.js

```diff
@@ -335,6 +336,11 @@ Addon.prototype.config = function (env, baseConfig) {
   }
 };
 
+Addon.prototype.dependencies = function(pkg) {
+  pkg = pkg || this.pkg || {};
```

Comment:
Ok, should it be something along the lines of `var package = pkg || this.pkg || {}`? I lifted this method from the EmberApp class and left it otherwise unchanged.